internal/actions/outputs.go

```diff
@@ -44,8 +46,11 @@ func setOutputs(outputs map[string]string) error {
 
 func printAndWriteString(f *os.File, out string) error {
 	fmt.Print(out)
-	if _, err := f.WriteString(out); err != nil {
-		return fmt.Errorf("error writing output: %w", err)
+	// We only want to print outputs if we are in test mode
```

Comment:
This comment seems wrong
Also why would we not want to print the outputs in test mode?

Reply:
Oh I think I just worded it confusingly.

We always print outputs. We don't actually write the outputs in test mode